style.css

```diff
@@ -85,11 +87,264 @@ ul {
+}
+
+.bottom::after {
+  position: relative;
+  left: 22px;
+  top: 8px;
+  content: '';
+  display: inline-block;
+  margin-top: 0.6em;
+  width: 1em;
+  height: 0.9em;
+  border-top: 0.2em solid #333;
+  border-right: 0.2em solid #333;
+  border-radius: 3px;
+  -moz-transform: rotate(135deg);
+  -webkit-transform: rotate(135deg);
+  transform: rotate(135deg);
+}
+
+.works-title {
+  font-size: 2.5rem;
+  text-align: center;
+  letter-spacing: 1px;
+  width: 80%;
+  font-weight: 900;
+  color: #172b4d;
+  margin: 3vh auto 5vh auto;
+}
+
+.works-title::after {
+  border-radius: 10px;
+  content: "";
+  width: 48px;
+  display: block;
+  height: 4px;
+  background-color: #32b37e;
+  margin: auto;
+  margin-top: 20px;
+}
+
+.projects-container {
+  display: grid;
+  grid-template-rows: auto;
+  column-gap: 10px;
+}
+
+.project {
+  background-color: #ebf0ee;
+  width: 90%;
+  margin: 2vh auto;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #d0d9d4;
+}
+
+.project-image {
+  overflow: hidden;
+  width: 100%;
+  height: 300px;
+  border-radius: 5px;
+  background-size: auto;
+  background-position: center;
+}
+
+.project-description {
+  border-radius: 10px;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.project-description h3 {
+  font-size: 1.5rem;
+  text-align: center;
+  letter-spacing: 1px;
+  font-weight: 900;
+  color: #3a4a42;
+  margin-top: 20px;
+}
+
+.project-description ul {
+  padding: 10px;
+  margin: auto;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+}
+
+.project-description ul > li {
+  font-size: 12px;
+  font-weight: 500;
+  margin: 5px 0;
+  padding: 10px;
+  background-color: #ebf0ee;
+  border-radius: 5px;
+}
+
+.project-description > .btn {
+  border-radius: 5px;
+  font-size: 1.3rem;
+  text-align: center;
+  text-decoration: none;
+  padding: 15px;
+  background-color: #32b37e;
+  color: white;
+  margin: 15px 0 25px 0;
+}
+
+.about-section {
+  display: block;
+}
+
+.about-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+  margin: auto;
+}
+
+.about-description {
+  text-align: center;
+  width: 90%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 20px;
+}
+
+.about-title {
+  font-size: 40px;
+  font-weight: 700;
+  text-align: center;
+  line-height: 52px;
+}
+
+.about-text {
+  /* height: 25vh; */
+  line-height: 32px;
+  font-size: 1.4rem;
+  text-align: center;
+  font-weight: 500;
+  color: #172b4d;
+  margin-top: 20px;
+  margin-bottom: 10px;
+  border: none;
+}
+
+.about-description > .btn {
+  border-radius: 5px;
+  font-size: 1.3rem;
+  text-align: center;
+  text-decoration: none;
+  padding: 15px;
+  background-color: #32b37e;
+  color: white;
+  margin: 15px 0 25px 0;
+}
```

Comment:
- [ ] Please, implement the button interactions (enable, hover, etc.), on the works section and about section, it is for a better user experience. ⭐ 
![image](https://user-images.githubusercontent.com/58642949/185606287-9100c049-bb4d-40c2-9492-f55d0a238922.png)